src/test/java/net/mcreator/integration/generator/GTProcedureBlocks.java

```diff
@@ -142,6 +142,19 @@ public static void runTest(Logger LOG, String generatorName, Random random, Work
 											.append(opt.get(1).getAsString()).append("</field>");
 									processed++;
 									break;
+								case "field_data_list_selector":
+									String type = arg.get("datalist").getAsString();
+									if (type.equals("enchantment"))
+										type = "enhancement";
+									String[] values = BlocklyJavascriptBridge.getListOfForWorkspace(workspace, type);
+									if (values.length > 0 && !values[0].equals("")) {
+										String value = type.equals("entity") ?
+												"EntityZombie" : ListUtils.getRandomItem(random, values);

```

Comment:
It may make sense to extend this system to support random entity at this point

Reply:
The issue is with the "Spawn entity" procedure blocks: right now there's no way to differentiate between spawnable and non-spawnable entities (e.g. `LivingEntity`)

Reply:
I see

Reply:
Hold on, if even I knew this was the reason, why didn't you remember about that thing, Klemen?

Reply:
It happens